gno.land/cmd/gnoland/testdata/cpu-cycle-overrun1.txtar

```diff
@@ -0,0 +1,64 @@
+loadpkg gno.land/p/demo/avl
```

Comment:
Can we run this txtar conditionally, so it doesn't hang in the local suite, but is present in the codebase? cc @gfanton

Reply:
We should not have failing tests in the codebase unless we have a system that expects them to fail and checks that they are still failing

Ie. what we should do for the "challenges" directory